docs/actions.md

```diff
@@ -598,6 +598,28 @@ You can clean up by deleting actions that you do not want to use.
   actions
   /guest/mySequence                private sequence
   ```
+## Protect action updated or deleted by mistake
```

Comment:
I will suggest some updates to this text once the API change and implementation are finalized.

Reply:
Updated accordingly